src/core/mark/legend.ts

```diff
@@ -6,10 +6,13 @@ import type { Dimension } from '../utils/position';
 import { scaleLinear, ScaleLinear } from 'd3-scale';
 import { getTextStyle } from '../utils/text-style';
 
+// Just the libraries necesssary fro this module
+type Libraries = Pick<import('@higlass/available-for-plugins').Libraries, 'PIXI' | 'd3Selection' | 'd3Drag'>;
```

Comment:
This is done for testing, and to avoid uses of `any` or `@ts-ignore` in our tests. 

Now that `HGC` is fully typed, our tests have type errors:

```typescript
drawMark({ libraries: { PIXI: /* ... */ } }, ...args)
                   // ^ type error, not a complete HGC type.
```

So I narrowed the type to just the features which are used for the module so that it is easier to test.

Reply:
Ah, gotcha!